start.js

```diff
@@ -0,0 +1,122 @@
+const http = require("http");
+const soldierRoute = require("./routes/soldiers.js");
+const dutiesRoute = require("./routes/duties");
+const justiceBoardRoute = require("./routes/justiceBoard");
+const configs = require('./config');
+const MongoClient = require('mongodb').MongoClient;
+const utils = require("./utility/utility");
+const soldierDb = require("./collections/soldiers");
+
+// Connection URL
+const mongoUrl = configs.dbUrl;
+
+// Database Name
+const dbName = configs.dbName;
+
+let db = 1;
+
+class ErrorHandler {
+  constructor(callback) {
+    this.callback = callback;
+  }
+
+  success(data) {
```

Comment:
this is not an error. so the class name is wrong. change it as to not be confusing